test/net/sourceforge/kolmafia/swingui/panel/PreferenceWatcherTableTest.java

```diff
@@ -165,9 +165,10 @@ void canRemoveProperty() {
               withProperty("testPrefA", "a"),
               withProperty("testPrefB", "b"),
               withProperty("watchedPreferences", "testPrefA,testPrefB"));
-
+      var table = new PreferenceWatcherTable();
```

Comment:
This is believed to fix a concurrent modification exception sometimes thrown in the test.